src/sidebar/components/ThreadCard.tsx

```diff
@@ -75,27 +77,23 @@ function ThreadCard({ frameSync, thread }) {
   return (
     /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
     <Card
-      classes={classnames('p-3 cursor-pointer focus-visible-ring', {
-        'is-hovered': isHovered,
-      })}
+      active={!!isHovered}
```

Comment:
I would suggest to coerce `isHovered` to a boolean at the point where it is declared if needed, rather than where it is used, as the name implies it is a boolean.

Reply:
That makes good sense, thanks.